src/main/java/com/gotocompany/depot/common/Template.java

```diff
@@ -19,8 +19,9 @@ public Template(String template) throws InvalidTemplateException {
             throw new InvalidTemplateException("Template cannot be empty");
         }
         List<String> templateStrings = new ArrayList<>();
-        Splitter.on(",").omitEmptyStrings().split(template).forEach(s -> templateStrings.add(s.trim()));
-        this.templatePattern = templateStrings.get(0);
+
+        Splitter.onPattern("(?<!\\\\\",),(?!\\\")").omitEmptyStrings().split(template).forEach(s -> templateStrings.add(s.trim()));
```

Comment:
The approach to escape comma \",\" looks good (ideal for programmers)

Can we consider it in simpler way /,/ by doing this we can make the escape literal smaller and also we can avoid the above two regular expressions? I'm not sure about the pitfalls of the suggested /,/ pattern, the intention is to have more user-friendly escape literal & not sure of performance impact of the above regular expressions which has look-behind & look-ahead regex. 
@sbchaos your thoughts?

Reply:
ok let me try this

Reply:
@rajuGT but the regular expression will still be required right? also performance wont be affected much , this template object is created only once during firehose starting

Reply:
done, ive changed the escape pattern to \,\